src/components/Image/Image.tsx

```diff
@@ -84,14 +97,27 @@ export class Image extends React.Component<ImageProps> {
       width,
       height,
       onLoad,
+      aspectRatio,
       resize,
       loadingBehavior,
       ...imageProps
     } = this.props;
     const { isLoaded } = this.state;
-    const dimensions = { width, height };
 
     const isAbsoluteUrl = src && src.match('^https?://');
+
+    const aspectRatioAsNumber =
```

Comment:
It's a bit odd to name this variable like that since aspectRatio is typically a number. But in our case it also supports the predefined values 😄

On the other hand, not sure I would rename the prop...

Reply:
I think that it is good to keep the prop name as is even it could also be a string :)